pkg/outputter.go

```diff
@@ -197,6 +197,13 @@ func parseConfigTemplates(ctx context.Context, c *config.Config, iface *Interfac
 		"outpkg":   &c.Outpkg,
 	}
 
+	// Functions available in the template for manipulating
+	// the individual key/value pairs from [templateMap] above
+	templateFuncMap := template.FuncMap{
+		"replace":    templateReplaceFn,
+		"trimPrefix": templateTrimPrefixFn,
+	}
```

Comment:
Why not reference `strings.ReplaceAll` directly etc? Let's also call the function name `replaceAll`.

Reply:
Most did it like this because

1. I hacked it up at 3 am, and I was tired 😆 
2. See (1)